<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="input_OP1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>261</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_OP1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OP1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_OP2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>90</y>
      <width>125</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>OP2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="summe_btn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>140</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_operationen">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>80</width>
      <height>272</height>
     </rect>
    </property>
    <property name="title">
     <string>Operation</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="plus_op">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>+</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="minus_op">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="mal_op">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="geteilt_op">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="summe_label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>631</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>259</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>113</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="input_OP2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>summe_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>myCalc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>summe_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>timclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>911</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>myCalc()</slot>
  <slot>timclick()</slot>
 </slots>
</ui>
